src/form/field/TextArea.mjs

```diff
@@ -126,6 +132,9 @@ class TextArea extends Text {
         }
 
         super.afterSetValue(value, oldValue);
+        if (this.autoGrow) {
+            this.syncAutoGrowHeight();
```

Comment:
Hi Nige, PR looks good in general. Just wondering about this spot: I think we need a `this.mounted` check, otherwise there would be  a DOM access on a non existing node. So we will also need `afterSetMounted()` => if true, call the sync method.